documentation/book/proc-customizing-deployments.adoc

```diff
@@ -0,0 +1,56 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='proc-customizing-deployments-{context}']
+= Customizing deployments
+
```

Comment:
_Add an introduction:_

This procedure describes how to customize a Kaka deployment using `Labels`. You can also use `Annotations`.